src/read.c

```diff
@@ -2566,6 +2669,63 @@ static avifBool avifParsePixelInformationProperty(avifProperty * prop, const uin
             return AVIF_FALSE;
         }
     }
+#if defined(AVIF_ENABLE_EXPERIMENTAL_EXTENDED_PIXI)
+    if (flags & 1) {
+        for (uint8_t i = 0; i < pixi->planeCount; ++i) {
+            uint8_t channelIdc, reserved, componentFormat, channelLabelFlag;
+            AVIF_CHECK(avifROStreamReadBitsU8(&s, &channelIdc, /*bitCount=*/3));      // unsigned int(3) channel_idc;
+            AVIF_CHECK(avifROStreamReadBitsU8(&s, &reserved, /*bitCount=*/1));        // unsigned int(1) reserved = 0;
+            AVIF_CHECK(avifROStreamReadBitsU8(&s, &componentFormat, /*bitCount=*/2)); // unsigned int(3) component_format;
+            AVIF_CHECK(avifROStreamReadBitsU8(&s, &pixi->subsamplingFlag[i], /*bitCount=*/1)); // unsigned int(3) subsampling_flag;
+            AVIF_CHECK(avifROStreamReadBitsU8(&s, &channelLabelFlag, /*bitCount=*/1)); // unsigned int(3) channel_label_flag;
+            if (pixi->subsamplingFlag[i]) {
+                AVIF_CHECK(avifROStreamReadBitsU8(&s, &pixi->subsamplingType[i], /*bitCount=*/4)); // unsigned int(4) subsampling_type;
+                AVIF_CHECK(avifROStreamReadBitsU8(&s, &pixi->subsamplingLocation[i], /*bitCount=*/4)); // unsigned int(4) subsampling_location;
+            }
+
+            // ISO/IEC 23008-12:2024/CDAM 2:2025 section 6.5.6.3:
+            //   This field indicates the contents of the channel. A value of 0 indicates colour/grayscale. A value of
+            //   1 indicates alpha. A value of 2 indicates depth. Values 3-7 are reserved for future use. At most one
+            //   channel shall have a channel_idc of 1.
+            if (channelIdc != 0) {
```

Comment:
this is just not implemented, right? (since it doesn't match the comment just above it)
ideally it should lead to a different return status.
(same for unsupported component_format below?)